tests/components/shelly/conftest.py

```diff
@@ -70,6 +77,7 @@
     "auth": False,
     "fw": "20201124-092854/v1.9.0@57ac4ad8",
     "num_outputs": 2,
+    "ver": "20201124-092854/v1.9.0@57ac4ad8",
```

Comment:
You should not mix properties between Gen1 & Gen2 devices.
Here is an example of Gen2 device `/shelly` endpoint:
```json
{
   "name":"Test Gen2",
   "id":"shellyplus2pm-123456789abc",
   "mac":"123456789ABC",
   "model":"SNSW-002P16EU",
   "gen":2,
   "fw_id":"20220830-130540/0.11.0-gfa1bc37",
   "ver":"0.11.0",
   "app":"Plus2PM",
   "auth_en":false,
   "auth_domain":null,
   "profile":"cover"
}
```
Please split into different mocks - `MOCK_BLOCK_SHELLY` & `MOCK_RPC_SHELLY`

Reply:
Done, I have split MOCK_SHELLY_COAP and MOCK_SHELLY_RPC